taskcluster/scripts/build/windows.ps1

```diff
@@ -26,7 +26,7 @@ git submodule update --remote i18n
 
 # Fix: pip scripts are not on path by default on tc, so glean would fail
 $PYTHON_SCRIPTS =resolve-path "$env:APPDATA\Python\Python36\Scripts"
-$env:PATH ="$FETCHES_PATH;$QTPATH;$PYTHON_SCRIPTS;$env:PATH"
+$env:PATH ="$FETCHES_PATH;$FETCHES_PATH/VisualStudio/VC/Tools/MSVC/14.30.30705/bin/Hostx64/x64;$env:PATH;$QTPATH;$PYTHON_SCRIPTS;$env:PATH"
```

Comment:
I dislike that this means we hardcode the msvc version number in yet more places. Given this doesn't actually seems to be necessary for this script and the qt one, do we really want to?

Reply:
I don't feel too strongly. I mainly added this because it seems like it would make things more deterministic, rather than relying on whatever is set on the system already?